### ISSUE TITLE
Sample cmakefile for personal projects seems to be missing an argument

### ISSUE THREAD

**Issue by jordigh** [artivis/manif#106]
If I'm reading this right,

https://cmake.org/cmake/help/v3.0/command/target_include_directories.html

this line in the readme

```cmake
target_include_directories(${PROJECT_NAME} SYSTEM  ${manif_INCLUDE_DIRS})
```

should probably be something like

```cmake
target_include_directories(${PROJECT_NAME} SYSTEM PUBLIC ${manif_INCLUDE_DIRS})
```

Without the `PUBLIC` directive I get

```
CMake Error at CMakeLists.txt:7 (target_include_directories):
  target_include_directories called with invalid arguments
```

**Comment by artivis** [comment]
You are right, the scope keyword is missing. Fixing it.

**Comment by artivis** [comment]
Fixed in 085c25b8444d36eac3cda1b3a7a0dc0c32e734bb.
